configs/crystal_defconfig

```diff
@@ -18,9 +18,14 @@ CONFIG_SYS_I2C_MXC_I2C3=y
 CONFIG_SYS_I2C_MXC_I2C4=y
 CONFIG_DEFAULT_DEVICE_TREE="imx6dl-crystal2"
 CONFIG_SPL_TEXT_BASE=0x00908000
-CONFIG_SPL_MMC_SUPPORT=y
-CONFIG_SPL_SERIAL_SUPPORT=y
+CONFIG_SPL_MMC=y
+CONFIG_SPL_SERIAL=y
 CONFIG_SPL=y
+CONFIG_SYS_SPL_MALLOC=y
```

Comment:
@esesyn01 why do we need it?

Reply:
this same with other configs under this. Are you sure that all of these are required to solve issues with eMMC booting?

Reply:
@kotylamichal Without them, u-boot build does not result in SPL file and so on. so we need them